src/auro-datetime.js

```diff
@@ -154,10 +156,29 @@ class AuroDatetime extends LitElement {
       toLowerCase();
   }
 
+  /**
+   * @private Internal function to generate proper time zone local.
+   * @returns {string} - Date/Time zone string.
+   * @param {boolean} returnType - Determines if to return date or time type.
+   */
+  tzTime(returnType) {
+
+    const scrubTimeZone = this.setDate.slice(0, -6);
+    const newDateTime = new Date(scrubTimeZone);
+
+    if (returnType) {
+      return newDateTime.toLocaleString('en-us', this.dateTemplate).replace(/^0+/u, '').
+        toLowerCase();
+    }
+
+    return newDateTime.toLocaleString('en-us', this.timeTemplate).replace(/^0+/u, '').
+      toLowerCase();
+  }
+
```

Comment:
What do you think about making this more explicit by passing in a string instead of a boolean? Would also require changes in `whichDate()` function.

```suggestion
  /**
   * @private Internal function to generate proper time zone local.
   * @returns {string} - Date/Time zone string.
   * @param {string} returnType - Determines if to return date or time type.
   */
  tzTime(returnType) {

    const scrubTimeZone = this.setDate.slice(0, -6);
    const newDateTime = new Date(scrubTimeZone);
    
    switch(returnType) {
      case 'date':
        return newDateTime.toLocaleString('en-us', this.dateTemplate).replace(/^0+/u, '').
          toLowerCase();
      case 'time':
      default:
        return newDateTime.toLocaleString('en-us', this.timeTemplate).replace(/^0+/u, '').
          toLowerCase();
    }
  }

```

Reply:
Looking at this again... really we can drop the whole if/then scenario. The return is identical other than the template used. So, having a single return and making the @param the template type, we get the same outcome and it is way more explicit.

Reply:
I like this even better.